packages/core/src/utils/resolve-from.ts

```diff
@@ -0,0 +1,6 @@
+import { resolvePath } from 'mlly';
```

Comment:
I think the dep on mlly has been deleted in the package.json?

Reply:
It's still [there](https://github.com/seek-oss/crackle/blob/vite-lib-mode/packages/core/package.json#L103), just synced it to Vite's version so the diff is a bit more noisy.